react-frontend-app/src/App.js

```diff
@@ -12,15 +12,18 @@ import './style/App.css';
 
 // Import different components
 import Description from './components/Description'
-import Display from './components/Display'
 import Footer from './components/Footer'
 import MainNavBar from './components/MainNavBar'
 import BarGraph from './components/BarGraph'
 
 function App() {
   const [surname, setSurname] = useState('');
   const [zipcode, setZip] = useState('');
-  var prediction = {};
+  const [loading, setLoading] = useState(true);
+
+  // var prediction = {};
+  var [variable, setVariable] = useState('this is the default variable');
```

Comment:
I'm really impressed with your command of React Hooks to manage the state variables even within a functional App.js! Great stuff

Reply:
+1 for this!